src/main/java/com/pirimid/cryptotrade/repository/OrderRepository.java

```diff
@@ -1,12 +1,16 @@
 package com.pirimid.cryptotrade.repository;
 
+import com.pirimid.cryptotrade.model.Account;
+import com.pirimid.cryptotrade.model.Exchange;
 import com.pirimid.cryptotrade.model.Order;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.Optional;
 import java.util.UUID;
 
 @Repository
 public interface OrderRepository extends JpaRepository<Order, UUID> {
-
+    Optional<Order> findByOrderIdExchangeAndAccount(String exchangeOrderId, Account account);
+    Optional<Order> findByOrderIdExchangeAndAccount_Exchange(String excOrderId, Exchange exchange);
```

Comment:
Can you elaborate a bit about this method?

Reply:
line 14
it will find an order on the basis of order_id generated by exchange and account (stored in our db)

line 15
it will find an order on the basis of of order_id generated by exchange and exchangee